alfasim_sdk/hook_specs.py

```diff
@@ -1333,6 +1333,82 @@ def calculate_entrained_liquid_fraction(
     """
 
 
+def update_internal_deposit_layer(
+    ctx: "void*", deposit_layer_thickness: "void*", n_control_volumes: "int"
+) -> "int":
+    """
+    **c++ signature** : ``HOOK_UPDATE_INTERNAL_DEPOSIT_LAYER(void* ctx, void* deposit_layer_thickness,
+    int n_control_volumes)``
+
+    Internal simulator hook to evaluate the thickness of the deposited layer at the inside of the pipeline walls.
+    This is called at the beginning of accounting the diameter reduction.
+
+    The plugin is supposed to change the given ``deposit_layer_thickness`` array pointer. Its values are contiguous
+    in memory and the dimension is given by ``n_control_volumes``. It has unit equal to ``[m]``.
+
+    :param ctx: ALFAsim's plugins context
+    :param deposit_layer_thickness: Thickness of the internal deposit layer
+    :param n_control_volumes: Number of control volumes
+    :returns: Return OK if successful or anything different if failed
+
+    Example of usage
+    .. code-block:: c++
+        :linenos:
+        :emphasize-lines: 1
+
+        HOOK_UPDATE_INTERNAL_DEPOSIT_LAYER(
+            ctx, deposition_layer_thickness, n_control_volumes)
+        {
+            auto errcode = -1;
+            void* deposit_layer_thickness_raw_ptr = nullptr;
+            errcode = alfasim.get_plugin_variable(
+                ctx, &deposit_layer_thickness_raw_ptr, "deposit_layer_thickness", 0, TimestepScope::CURRENT, &size);
+            if (errcode != 0) {
+                return errcode;
+            }
+            auto* deposit_layer_thickness = (double*) (deposit_layer_thickness_raw_ptr);
```

Comment:
Can you explain why this code block is necessary, since `deposition_layer_thickness` is being passed as parameter?
What is the difference between `deposition_layer_thickness` and `deposit_layer_thickness`?

Reply:
I'm sorry `deposition_layer_thickness` and `deposit_layer_thickness` are the same, just with a typo and I don't need this block, this was probably a copy and paste

Reply:
- [ ] remove get_plugin_variable for deposit_layer_thickness